src/detect/ScanI2C.cpp

```diff
@@ -41,6 +41,12 @@ ScanI2C::FoundDevice ScanI2C::firstAccelerometer() const
     return firstOfOrNONE(8, types);
 }
 
+ScanI2C::FoundDevice ScanI2C::firstAQI() const
+{
+    ScanI2C::DeviceType types[] = {PMSA0031, SCD4X};
+    return firstOfOrNONE(2, types);
+}
+
```

Comment:
I understand the intention. The PMSA003I measures the particles and the SCD4x CO2. When connecting both, only the values of the PMSA003I are measured and the SCD4x is ignored. Maybe differ between particles and co2, so that particles and co2 can be measured. Such an approach could be used if for example an SCD30 and SCD4x is connected.

Reply:
Generally meshtastic only supports one sensor per type at a time.